dask_searchcv/tests/test_model_selection.py

```diff
@@ -620,3 +620,63 @@ def test_scheduler_param_distributed(loop):
 def test_scheduler_param_bad():
     with pytest.raises(ValueError):
         _normalize_scheduler('threeding', 4)
+
+
+class CacheCVLike(CVCache):
+    called = 0
+    def __init__(self, splits=None, pairwise=False, cache=True):
```

Comment:
Is this init doing anything other than calling the parent?

Reply:
`splits` is a required argument in `CVCache.__init__`: https://github.com/dask/dask-searchcv/blob/master/dask_searchcv/methods.py#L68 but it must be a `splits=None` in the signature, otherwise I can't instantiate the `CVCacheLike` instance (`splits` are determined inside the the fitting logic and then the `cache.splits = splits` happens in `cv_split` as in our discussion above.